styles.css

```diff
@@ -0,0 +1,491 @@
+  border-bottom: 1px solid #dfe1e6;
+  margin-bottom: 24px;
+}
+
+.lists h4 {
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-size: 20px;
+  font-weight: 500;
+}
+
+/* contactme */
+
+.footer-container {
+  background-color: #fff;
+  padding-top: 117px;
+}
+
+.contact-me {
+  margin: auto;
+  padding-left: 24px;
+  padding-right: 24px;
+  background: #6070ff;
+  background-image: url(assets/images/desktop-images/contact\ form\ background\ shapes@2x.svg);
+  background-repeat: no-repeat;
+  background-position-x: right;
+  border-top-left-radius: 90px;
+  padding-top: 103px;
+}
+
+.footer-container h2 {
+  color: #fff;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+}
+
+.contact-me p {
+  font-family: Poppins, serif, sans serif;
+  font-weight: 200;
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+  padding-top: 12px;
+}
+
+.contact-me p,
+.contact-me h2 {
+  text-align: center;
+}
+
+input {
+  border-radius: 0;
+  border-radius: 8px;
+  padding: 15px 38px 15px 16px;
+  border: 1px solid #cfd8dc;
+  margin-top: 20px;
+  font-family: Poppins, serif, sans serif;
+  font-weight: 400px;
+  color: #172b4d;
+  font-size: 17px;
+}
+
+textarea {
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  margin-top: 18px;
+  padding: 15px 38px 15px 16px;
+  font-family: Poppins, serif, sans serif;
+  font-weight: 400px;
+  color: #172b4d;
+  font-size: 17px;
+}
+
+form input,
+form textarea {
+  display: block;
+  width: 100%;
+}
+
+.get-in-touch-btn {
+  font-family: "Poppins", sans-serif;
+  background-color: #fff;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  color: #396df2;
+  padding: 12px;
+  text-align: left !important;
+  text-decoration: none;
+  cursor: pointer;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  margin-top: 15px;
+  letter-spacing: 0.03em;
+  transition: background-color 0.15s, color 0.15s;
+}
+
+.get-in-touch-btn:hover {
+  color: #fff;
+  background-color: #2230d2;
+  border: 2px solid #fff;
+}
+
+.get-in-touch-btn:active {
+  color: #fff;
+  background-color: #6070ff;
+}
+
+.end-bar {
+  display: flex;
+  justify-content: center;
+  margin: 5% 3%;
+}
+
+/* Desktop  */
+@media screen and (min-width: 768px) {
+  .hero {
+    background-image: url(assets/images/desktop-images/Header\ bg.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    padding: 120px 260px 435px 260px;
+  }
+
+  header {
+    padding: 25px 155px 25px 155px;
+  }
+
+  .nav-menu {
+    display: flex;
+  }
+
+  .works {
+    margin: 142px 142px 142px 142px;
+  }
+
+  .cards-description {
+    margin-top: 30px;
+  }
+
+  .burger {
+    display: none;
+  }
+
+  .nav-item {
+    font-family: Poppins, serif, sans serif;
+    padding-left: 32px;
+    color: #344563;
+  }
+
+  .cards {
+    display: grid;
+    gap: 20px;
+    grid-template-columns: 1fr 1fr;
+  }
```

Comment:
- On the desktop view, you are supposed to add a hover effect on the card, as indicated on the Figma design, please add the hover effect properties as it is on the Figma.

##### Figma Deign to show hove effect (The blue border color around it)
```css
border: 2px solid #A7AEFF;
```

![image](https://github.com/znation2010/Mobile_Version_Portfolio/assets/58771507/534788e3-e877-4981-bcbf-f2e2a48d0bd0)